### PR TITLE
chore: shape conversion tools

### PR DESCRIPTION
**Pull request by jeswr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 - [RDFminer](https://github.com/Wimmics/RDFminer) - Web application to automatically discover SHACL shapes representative of an RDF data graph, by Wimmics; `CECILL-C` license; `Java`.
 - [SHACL Discovery Service](https://github.com/AKSW/discover-shacl-shapes) - Example implementation of a discovery service for SHACL shapes/shape groups; `MIT` license; `PHP`.
 - [Shapes of You index](https://index.semanticscience.org/) - SPARQL queries, OWL/SKOS vocabularies, SHACL/ShEx shapes, indexed from public `git` repositories; `MIT` license; `Typescript`, `Python`.
+- [SHACL Play!](https://shacl-play.sparna.fr/play/shapes-catalog) - To see your shapes file listed here, add it to the Shapes Catalog source file on Github.
```
Comment (TallTed) [inline README.md:64]: Include a direct link to the "Shapes Catalog source file on Github"?

**Comment by VladimirAlexiev** [comment]
@jeswr , what does "chore" mean in this context?

**Comment by jeswr** [comment]
> @jeswr , what does "chore" mean in this context?

Yeah, that probably should be `feat:`. I was treating this as a documentation update and following [semantic release commit messaging](https://github.com/semantic-release/semantic-release)

File: README.md [hunk 2]
```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL. 
 - [owl2shacl](https://github.com/sparna-git/owl2shacl) - OWL-to-SHACL conversion rules.
+- [TopQuadrant: OWL Axioms to SHACL Constraints](https://www.topquadrant.com/doc/latest/reference/Transform_ConvertOWLAxiomsToSHACLConstraints.html) - Commercial closed-source solution by TopQuadrant
+- [elevont/owl2shacl](https://github.com/elevont/owl2shacl) - 
+[![License: AGPL-3.0-or-later](
```
Comment (VladimirAlexiev) [inline README.md:81]: are we using images for licenses? All others use something like "`AGPL` license".

File: README.md [hunk 3]
```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL. 
 - [owl2shacl](https://github.com/sparna-git/owl2shacl) - OWL-to-SHACL conversion rules.
+- [TopQuadrant: OWL Axioms to SHACL Constraints](https://www.topquadrant.com/doc/latest/reference/Transform_ConvertOWLAxiomsToSHACLConstraints.html) - Commercial closed-source solution by TopQuadrant
```
Comment (VladimirAlexiev) [inline README.md:79]: Use this icon: 💰 : Denotes closed source commercial software.

File: README.md [hunk 4]
```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL.
```
Comment (VladimirAlexiev) [inline README.md:77]: - change name to "SHACL Play! Convertor"
- Shorten descr to "Converts between RDF schema formats. Has OWL-to-SHACL (open, semi-closed, closed)"

File: README.md [hunk 5]
```diff
@@ -61,6 +61,7 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 - [RDFminer](https://github.com/Wimmics/RDFminer) - Web application to automatically discover SHACL shapes representative of an RDF data graph, by Wimmics; `CECILL-C` license; `Java`.
 - [SHACL Discovery Service](https://github.com/AKSW/discover-shacl-shapes) - Example implementation of a discovery service for SHACL shapes/shape groups; `MIT` license; `PHP`.
 - [Shapes of You index](https://index.semanticscience.org/) - SPARQL queries, OWL/SKOS vocabularies, SHACL/ShEx shapes, indexed from public `git` repositories; `MIT` license; `Typescript`, `Python`.
+- [SHACL Play!](https://shacl-play.sparna.fr/play/shapes-catalog) - To see your shapes file listed here, add it to the [Shapes Catalog source file on Github](https://github.com/sparna-git/SHACL-Catalog/blob/master/shacl-catalog.ttl).
```
Comment (VladimirAlexiev) [inline README.md:64]: change link name to "SHACL Play! Catalog"

File: README.md [hunk 6]
```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
...
+    https://img.shields.io/crates/v/owl2shacl.svg)](
+    https://crates.io/crates/owl2shacl)
+A CLI tool that tries to convert simple OWL ontologies into SHACL shapes.
 
 ## Shape-based Query Generators
 
 - [shape-to-query](https://github.com/hypermedia-app/shape-to-query) <img alt="NPM Version" src="https://img.shields.io/npm/v/@hydrofoil/shape-to-query" align="top"> <img alt="GitHub Release Date" src="https://img.shields.io/github/release-date/hypermedia-app/shape-to-query" align="top"> - Generate SPARQL queries from SHACL Shapes; [docs](https://shape-to-query.hypermedia.app/docs); `MIT` license; `Typescript`.
-  - [playground](https://shape-to-query.hypermedia.app)
+  - [playground](https://shape-to-query.hypermedia.app) - A CLI tool that tries to convert simple OWL ontologies into [SHACL] shapes.
+- [SHACL Play!](https://shacl-play.sparna.fr/play/sparql) - This is a SHACL-based SPARQL generator. It generates SPARQL queries used to extract a subset of data from a knowledge graph, based on the SHACL specification of the target dataset structure.
```
Comment (VladimirAlexiev) [inline README.md:93]: Change to 
SHACL Play! SPARQL: generates SPARQL queries to extract RDF subset based on SHACL specification of the target dataset.

File: README.md [hunk 7]
```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
...
+[![Package Releases](
+    https://img.shields.io/crates/v/owl2shacl.svg)](
+    https://crates.io/crates/owl2shacl)
+A CLI tool that tries to convert simple OWL ontologies into SHACL shapes.
 
 ## Shape-based Query Generators
 
 - [shape-to-query](https://github.com/hypermedia-app/shape-to-query) <img alt="NPM Version" src="https://img.shields.io/npm/v/@hydrofoil/shape-to-query" align="top"> <img alt="GitHub Release Date" src="https://img.shields.io/github/release-date/hypermedia-app/shape-to-query" align="top"> - Generate SPARQL queries from SHACL Shapes; [docs](https://shape-to-query.hypermedia.app/docs); `MIT` license; `Typescript`.
-  - [playground](https://shape-to-query.hypermedia.app)
+  - [playground](https://shape-to-query.hypermedia.app) - A CLI tool that tries to convert simple OWL ontologies into [SHACL] shapes.
```
Comment (VladimirAlexiev) [inline README.md:92]: The description is completely wrong, remove
- a web playground is not a CLI
- it is not OWL to SHACL but SHACL to SPARQL
- `[SHACL]` in brackets?

**Review by VladimirAlexiev — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (jeswr) [reply README.md:64]: When making proposals like this in the future I would suggest using the add a suggestion button in the reviewer so you can propose a diff that the reviewee can commit directly. Helps speed up the process for everyone.

Screenshot 2024-12-03 at 10 11 43

Comment (jeswr) [reply README.md]: See https://github.com/w3c-cg/awesome-semantic-shapes/pull/14#discussion_r1867433354 for what this looks like.

**Pull request merged**